js/views/itemShortVw.js

```diff
@@ -41,7 +43,23 @@ module.exports = Backbone.View.extend({
       }
 
       this.model.set('ownFollowing', false);
-    });    
+    });
+
+    this.listenTo(window.obEventBus, "blockingUser", function(e){
+      if (e.guid !== this.model.get('guid')) {
+        return;
+      }
+
+      this.model.set('isBlocked', true);
```

Comment:
It's not an issue either way, I was just wondering if something clever was going on I didn't notice.